tools/build/binary-release/Release_build_instructions.txt

```diff
@@ -0,0 +1,51 @@
+Windows

```

Comment:
Why `.txt`? What about `.md`?

Reply:
Also maybe use `-` instead of `_` in filenames (I'd guess most files do that)

Reply:
Done. I have left the underscores to match the release_guide.pod naming and moved the file to the `docs/` folder.